signal.c

```diff
@@ -520,6 +520,38 @@ rb_f_kill(int argc, const VALUE *argv)
     return INT2FIX(i-1);
 }
 
+/*
+ *  call-seq:
+ *     Process.kill(pid)    -> boolean
+ *
+ * Checks the given process to see whether it is alive.
+ */
+VALUE
+rb_f_exists(VALUE self, VALUE val_pid)
```

Comment:
I stuck this in `signal.c` just because it is a wrapper around kill, which is also in `signal.c`. I've now moved it to `process.c`. I'll move it back if it is decided that `signal.c` is the right place for it.